src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactAgeScoreTest.java

```diff
@@ -0,0 +1,103 @@
+package com.sap.oss.phosphor.fosstars.model.score.oss;
+
+import static com.sap.oss.phosphor.fosstars.TestUtils.assertScore;
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.RELEASED_ARTIFACT_VERSIONS;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
+
+import com.sap.oss.phosphor.fosstars.model.Score;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.value.ArtifactVersion;
+import com.sap.oss.phosphor.fosstars.model.value.ArtifactVersions;
+import com.sap.oss.phosphor.fosstars.model.value.ScoreValue;
+import java.time.LocalDate;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ArtifactAgeScoreTest {
+
+  @Test
+  public void smokeTest() {
+    long monthMilliseconds = 2629800000L;
+    ArtifactVersion version100 =
+        new ArtifactVersion("1.0.0", LocalDate.now().minusMonths(14));
+    ArtifactVersion version110 =
+        new ArtifactVersion("1.1.0", LocalDate.now().minusMonths(2));
+    ArtifactVersion version120 = new ArtifactVersion("1.2.0", LocalDate.now().minusDays(7));
+
+    assertScore(
+        Score.INTERVAL,
+        new ArtifactAgeScore(),
+        setOf(
+            RELEASED_ARTIFACT_VERSIONS.value(
+                ArtifactVersions.of(version100, version110, version120))
+        ));
+  }
+
+  @Test
+  public void currentReleasedVersion() {
+    ArtifactVersion version100 =
+        new ArtifactVersion("1.0.0", LocalDate.now().minusMonths(14));
+    ArtifactVersion version110 =
+        new ArtifactVersion("1.1.0", LocalDate.now().minusMonths(2));
+    ArtifactVersion version120 = new ArtifactVersion("1.2.0", LocalDate.now().minusDays(7));
+
+    ArtifactAgeScore score = new ArtifactAgeScore();
+    Value<ArtifactVersions> versions = RELEASED_ARTIFACT_VERSIONS.value(
+        ArtifactVersions.of(version100, version110, version120));
+    ScoreValue value = score.calculate(versions);
+    System.out.println("\n\ncurrentReleasedVersion");
+    System.out.printf("Available versions: %s;%nrating: %2.2f%n", versions, value.get());
+    Assert.assertEquals(Double.valueOf(10.0), value.get());
```

Comment:
This way it works `Assert.assertEquals(Double.valueOf(Score.MAX), value.get());`

Reply:
I am not sure why we need to convert it to `Double`. Any idea?

What kind of precision does it use here? Would it be better to set the precision instead?

`Assert.assertEquals(Score.MAX, value.get(), 0.01);`

If I remember correctly, there were some unstable test cases that didn't specify precision.

Reply:
> I am not sure why we need to convert it to Double. Any idea?

Problem is that the compiler can not decide if the first `double` should be autoboxed or the second `Double` unboxed.
Because the `assertEquals` exists for `assertEquals(double, double)` and `assertEquals(Double, Double)`.

Reply:
The `ArtifactLatestReleaseAgeScore` always produce natural numbers (and no fractional numbers).
At least in the current implementation.
Hence I would not add precision.

Reply:
`assertEquals(double, double)` seems to be deprecated

http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(double,%20double)

Let's please avoid using deprecated methods. You can use `Double.compare()` instead

https://docs.oracle.com/javase/8/docs/api/java/lang/Double.html#compare-double-double-

Reply:
The deprecated `assertEquals(double, double)` is not used.
Based on the reason for deprecation of `assertEquals(double, double)` we should either use `assertEquals(double, double, double)` with delta set or we can use `assertEquals(Double, Double)` which is not deprecated and use the `Double.equals()` at the end (which internally does the same as `Double.compare()`.

Based on the options and the test I would prefer to use `assertEquals(Double, Double)` as it is best readable does the same equal check as `Double.compare()`.

Reply:
Okay.